### PR TITLE
OpenApi: Fix params mapping on relative URL

### PR DESCRIPTION
**Pull request by fatal10110**
Fix for a bug, when `stripBasePath` is disabled and openApi is configured. it will not map path params to fastify params.

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: test/spec/openapi/route.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ const yaml = require('js-yaml')
 const fastifySwagger = require('../../../index')
 const {
   openapiOption,
+  openapiRelaiveOptions,
```
Comment (mcollina) [inline test/spec/openapi/route.js:10]: why this change?

File: examples/options.js [hunk 2]
```diff
@@ -250,6 +285,7 @@ const schemaOperationId = {
 
 module.exports = {
   openapiOption,
+  openapiRelaiveOptions,
```
Comment (mcollina) [inline examples/options.js:288]: what does relaive mean? Is it a typo?

**Review by mcollina — COMMENTED** [review]
I think the code should be linted.

File: examples/options.js (hunk 2 above)
Comment (fatal10110) [reply examples/options.js:288]: Thanks, will fix it

File: test/spec/openapi/route.js (hunk 1 above)
Comment (fatal10110) [reply test/spec/openapi/route.js:10]: Im using it in a test to, these are the options im able to reproduce the bug, so I thought this is the best way to test it

**Comment by fatal10110** [comment]
> I think the code should be linted.

Not sure what do you mean by "linted", I used `npm run test` which is also triggering the "standard" cli, also the CI checks passed

File: lib/spec/openapi/utils.js [hunk 3]
```diff
@@ -67,7 +67,7 @@ function prepareOpenapiObject (opts) {
 }
 
 function normalizeUrl (url, servers, stripBasePath) {
-  if (!stripBasePath) return url
+  if (!stripBasePath) return formatParamUrl(url)
```
Comment (climba03003) [inline lib/spec/openapi/utils.js:70]: Good catch

**Comment by mcollina** [comment]
> Not sure what do you mean by "linted", I used npm run test which is also triggering the "standard" cli, also the CI checks passed

nevermind, things were scrambled on mobile.

**Review by Eomm — APPROVED** [review]

**Review by climba03003 — APPROVED** [review]
